kmcd-prelude.md

```diff
@@ -16,29 +16,18 @@ module KMCD-PRELUDE
  // ---------------------------------------------------------------------
     rule DEPLOY-PRELUDE
       =>
-         // Contract Authorizations
-         // -----------------------
-
-         // Auhthorize Pot/End/Spot for Vat
-         transact ADMIN Vat . rely Pot
-         transact ADMIN Vat . rely End
-         transact ADMIN Vat . rely Spot
-
-         // Authorize End for Pot
-         transact ADMIN Pot . rely End
-
-         // Auhthorize Pot/End for Vow
-         transact ADMIN Vow . rely Pot
-         transact ADMIN Vow . rely End
+         // Main Deployments
+         // ----------------
 
-         // Authorize End for Cat/Pot
-         transact ADMIN Cat . rely End
+         transact ADMIN Gem "MKR" . init
+         transact ADMIN Deploy . deploy Gem "MKR"
 
-         // Authorize Vow for Flap
-         transact ADMIN Flap . rely Vow
+         transact ADMIN GemJoin "MKR" . constructor Vat Gem "MKR"
+         transact ADMIN Gem     "MKR" . initUser GemJoin "MKR"
+         transact ADMIN Deploy . deployCollateral "MKR" GemJoin "MKR"
 
-         // Authorize Vow for Flop
-         transact ADMIN Flop . rely Vow
+         transact ADMIN Gem "MKR" . initUser Vow
```

Comment:
I don't think the `Vow` needs an MKR balance? (line below is needed for the `Flap` for sure, though)

Reply:
Looks like for `lucash-flap-end` attacks, it's needed (because we do some hacky stuff to setup the attack state).

So I've localized this call to those attack files and removed it from this prelude.